rover-apps/pdb/src/PDB_Module.cpp

```diff
@@ -0,0 +1,114 @@
+#include "PDB_Module.h"
+
+#include <cmath>
+
+#include "Logger.h"
+#include "PDB_config.h"
+#include "hw_bridge.h"
+
+/* Pins configuration for Load Monitoring */
+DigitalOut load1_5V_diag_en(LOAD1_5V_DIAG_EN);
+DigitalIn load1_5V_fault_n(LOAD1_5V_FAULT);
+
+DigitalOut load2_5V_diag_en(LOAD2_5V_DIAG_EN);
+DigitalIn load2_5V_fault_n(LOAD2_5V_FAULT);
+
+DigitalOut load3_5V_diag_en(LOAD3_5V_DIAG_EN);
+DigitalIn load3_5V_fault_n(LOAD3_5V_FAULT);
+
+DigitalOut load4_5V_diag_en(LOAD4_5V_DIAG_EN);
+DigitalIn load4_5V_fault_n(LOAD4_5V_FAULT);
+
+DigitalOut load5_5V_diag_en(LOAD5_5V_DIAG_EN);
+DigitalIn load5_5V_fault_n(LOAD5_5V_FAULT);
+
+DigitalOut load_17V_diag_en(LOAD_17V_DIAG_EN);
+DigitalIn load_17V_fault_n(LOAD_17V_FAULT);
+
+/* Pins configuration for Rail Monitoring */
+AnalogIn railBattery(RAIL_BATTERY_ANLG_IN);
+AnalogIn rail5V(RAIL_5V_ANLG_IN);
+AnalogIn rail17V(RAIL_17V_ANLG_IN);
+AnalogIn rail24V(RAIL_24V_ANLG_IN);
+DigitalIn rail24V_pgood_n(RAIL_24V_PGOOD_N);
+
+/* Pins configuration for Temperature Monitoring */
+AnalogIn temperatureADC(TEMPERATURE_ADC_IN);
+
+PDB_Module::PDB_Module() {
+  load1_5V_diag_en = PDB_5V_LOAD1_DIAG_EN;
+  load2_5V_diag_en = PDB_5V_LOAD2_DIAG_EN;
+  load3_5V_diag_en = PDB_5V_LOAD3_DIAG_EN;
+  load4_5V_diag_en = PDB_5V_LOAD4_DIAG_EN;
+  load5_5V_diag_en = PDB_5V_LOAD5_DIAG_EN;
+  load_17V_diag_en = PDB_17V_LOAD_DIAG_EN;
+
+  rail_counter = 0;
+}
+
+void PDB_Module::load_monitoring() {
+  if (load1_5V_diag_en && !load1_5V_fault_n) {
+    Utility::logger << "Fault on 5V load 1\n";
```

Comment:
Can you leave a `//TODO` comment somewhere reminding to replace all printfs with CAN logs later?